native/components/OwnerView/OwnerView.jsx

```diff
@@ -0,0 +1,61 @@
+import { Pressable, StyleSheet, Text, View } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { useEffect, useState } from 'react';
+import { getAllUsers } from '../../services/api/users';
+
+export function OwnerView({ project }) {
+  const [applicants, setApplicants] = useState([]);
+  const navigation = useNavigation();
+
+  const showUser = (applicantID) => {
+    navigation.navigate('UserProfile', {
+      userID: applicantID,
+    });
+  };
+
+  const showApplicants = (id) => {
+    navigation.navigate('Applicants', {
+      projectID: id,
+    });
+  };
+  useEffect(() => {
+    const getApplicants = async () => {
+      if (project.interested_applicants?.length) {
+        const interested = await getAllUsers();
+        setApplicants(
+          interested.filter((user) => project.interested_applicants.includes(user._id))
+        );
+      }
+    };
+    getApplicants();
+  }, []);
+
+  return (
+    <View>
+      {project.interested_applicants?.length ? (
+        <View>
+          <Text>These users are interested in joining the project:</Text>
+          {applicants.slice(0, 3).map((applicant) => (
+            <Pressable key={applicant._id} onPress={() => showUser(applicant._id)}>
+              <Text
+                style={styles.applicant}
+              >{`${applicant.first_name} ${applicant.last_name}, ${applicant.role}`}</Text>
+            </Pressable>
+          ))}
+          {project.interested_applicants.length > 3 ? (
+            <Pressable onPress={() => showApplicants(project._id)}>
+              <Text>{`See all ${project.interested_applicants.length} applicants`}</Text>
+            </Pressable>
+          ) : null}
+        </View>
+      ) : null}
+    </View>
```

Comment:
I don't know how you managed to make what could have easily been a very funky looking block of code end up SO good looking. I'm taking notes 📝 

I'm curious if we have a preference for ternary's with trailing nulls or guard operators for circumstances like this one? I usually tend to use a guard operator to eliminate the extra null, but I kind of like the readability of the ? here and you managed to keep the null's not looking too out of place.

Reply:
I went ahead and changed (most of) the ternary statements with trailing `null` and replaced with the && operator; I think it reads a little more cleanly.